src/NUnitConsole/nunit3-console/EnginePackageSettings.cs

```diff
@@ -21,43 +21,16 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-namespace NUnit.Common
+namespace NUnit.Engine
```

Comment:
If we don't want to maintain the NUnit.Common namespace, then I would add these to the NUnit namespace.